modules/social_features/social_follow_user/social_follow_user.module

```diff
@@ -151,7 +154,14 @@ function social_follow_user_theme(): array {
  * Implements hook_preprocess_HOOK().
  */
 function social_follow_user_preprocess_profile(array &$variables): void {
-  if ($variables['elements']['#view_mode'] === 'teaser') {
+  // Skip any action if "follow user" is not active.
+  if (!\Drupal::config('social_follow_user.settings')->get('status')) {
+    return;
+  }
+
+  $view_mode = $variables['elements']['#view_mode'];
+
+  if ($view_mode === 'teaser') {
```

Comment:
@volodymyr-sydor Do we really need to add conditions for specific view modes?
Isn't it better to check for all possible view modes?

Reply:
We use follow functionality in specific view modes and if we add it then it will be available

Reply:
Ok